src/DBConnection2.cpp

```diff
@@ -0,0 +1,32 @@
+#include "DBConnection2.hpp"
+#include "ConnectionDispatcher.hpp"
+
+MySqlDBConnection2::MySqlDBConnection2(std::string serverVersion, int protocolVersion) {
+    m_info.serverVersion = std::move(serverVersion);
```

Comment:
@Morheit , unit members in in the init list, not in ctor body.

Reply:
@malirod , done